shared/pages/import/js/curtain.js

```diff
@@ -6,6 +6,13 @@ var Curtain = (function() {
 
   var _ = navigator.mozL10n.get;
 
+  if (!parent.document.querySelector('#fb-curtain')) {
```

Comment:
Add a bug to remove this `#fb-` prefix (sync with @mancas about this) and add a `TODO` comment in order to remove all these dependencies as next step.